dashboards-observability/.cypress/integration/VisualizationCharts/1_verticalBar_chart.spec.js

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// <reference types="cypress" />
```

Comment:
Rename this file with using proper underscores. Eg: 1_vertical_bar_chart.spec.js

Reply:
Renamed